CodeGeneration/Sources/generate-swift-syntax/templates/swiftsyntax/SyntaxNodesFile.swift

```diff
@@ -22,24 +22,12 @@ import Utils
 func syntaxNode(nodesStartingWith: [Character]) -> SourceFileSyntax {
   SourceFileSyntax(leadingTrivia: copyrightHeader) {
     for node in SYNTAX_NODES.compactMap(\.layoutNode) where nodesStartingWith.contains(node.kind.syntaxType.description.first!) {
-      let documentationSections = [
```

Comment:
While this is clean, there's a way to make this even nicer — we can rename `Node.documentation` to `Node.documenationSummary` and make a computed property `documentation` that works like `Child.documentation` and internally joins them together.

I'll push that change in a bit to this PR.

Reply:
I agree. In particular, it would be nice to be consistent with how we stitch together the comment sections.

Reply:
If we agree on this, then perhaps I should include that cleanup in this PR. 

I'm hesitant to bring new abstractions, or even move things into `Grammar` yet — I want to understand it's place better. But what we can do, is we can absolutely make `Node` and `Child` consistent with each other. I'm clear on how to do that, and have everything to prepare a commit.